src/pages/Lending/Pool/components/Repay/RepayInput.tsx

```diff
@@ -503,6 +503,28 @@ export const RepayInput = ({
     translate,
   ])
 
+  const quoteErrorTooltipTranslation = useMemo(() => {
```

Comment:
This might be tidier inside a `switch true` block, but this is totally fine too.